AVFoundation-CombineTests/TestSubscriber.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TestSubscriber.swift
+//  AVFoundation-CombineTests
+//
+//  Created by József Vesza on 2020. 07. 24..
+//  Copyright © 2020. József Vesza. All rights reserved.
+//
+
+import Combine
+
+class TestSubscriber<T>: Subscriber {
+    typealias Input = T
+    typealias Failure = Never
+    
```

Comment:
```suggestion
    /// Customizable demand this `Subscriber` will request from a `Subscription`.
```

Reply:
Added a slightly tweaked version of this